main.py

```
@@ -6,22 +6,32 @@
 app.add_middleware(BaseHTTPMiddleware, dispatch=log_request)
 
 app.include_router(frame_router, tags=["진단용 이미지 저장"])
+app.include_router(diagnosis_router, tags=["진단 결과 조회"])
 
 app.add_exception_handler(RequestValidationError, validation_exception_handler)
 app.add_exception_handler(StarletteHTTPException, http_exception_handler)
 app.add_exception_handler(Exception, generic_exception_handler)
 
+
+@app.on_event("startup")
+def startup_event():
```

Ruff (0.11.9)

Missing return type annotation for public function `startup_event`

Add return type annotation: `None`

(ANN201)